src/Resource/ServiceException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @package Nexcess-SDK
+ * @license https://opensource.org/licenses/MIT
+ * @copyright 2018 Nexcess.net, LLC
+ */
+
+declare(strict_types  = 1);
+
+namespace Nexcess\Sdk\Resource;
+
+use Nexcess\Sdk\Exception;
+
+class ServiceException extends Exception {
+  const NOT_CANCELLABLE = 1;
```

Comment:
This should have a docblock with a brief description of the error condition, like maybe
```php
/** @var int Attempt to cancel a service that is not cancellable. */
```